api/v3/ReportTemplate.php

```diff
@@ -109,7 +109,7 @@ function _civicrm_api3_report_template_getrows($params) {
   }
 
   $class = civicrm_api3('option_value', 'getvalue', array(
-    'option_group_id' => 'report_template',
```

Comment:
why was this changed? was if preference or because it stopped working? The api should continue to support strings for option_group_id so hopefully it was the former